hack/samples/apps/mysql.yaml

```diff
@@ -0,0 +1,208 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: mysql-todolist
+objects:
+  # TODO need VolumeSnapshotClass with label velero.io/csi-volumesnapshot-class: 'true'?
+  - kind: ServiceAccount
+    apiVersion: v1
+    metadata:
+      name: ${NAMESPACE}-sa
+      namespace: ${NAMESPACE}
+      labels:
+        component: ${NAMESPACE}
+  - kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: mysql
+      namespace: ${NAMESPACE}
+      labels:
+        app: mysql
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 1Gi
+  # - kind: SecurityContextConstraints
```

Comment:
can this be deleted @weshayutin ? A non admin user can not create that

example error
```
Error from server (Forbidden): error when creating "STDIN": securitycontextconstraints.security.openshift.io is forbidden: User "batman" cannot create resource "securitycontextconstraints" in API group "security.openshift.io" at the cluster scope
```

Reply:
yes, it can be deleted

Reply:
done